toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -348,6 +348,18 @@ please refer to the [overlay type](#overlay-type) section.
 - `hashPartition`: A partition used exclusively for storing a calculated hash
   tree.
 
+- `corruptionOption`: Optional. Specifies the behavior in case of detected
+  corruption. This is configurable with the following options:
+  - `CorruptionOptionDefault`: Without specifying corruption behaviors, or
```

Comment:
Do we need to expose that option? 
IMO, options should be `fail`, `ignore`, `panic` and `restart`. Not specifying any would pick the default which is 'fail' that we should clearly document here.

Reply:
also no need to put `CorruptionOptionXXX` here, just refer to the options directly

Reply:
Fixed, thanks.